### ISSUE TITLE
InterfacesAdded and someInterface.on()

### ISSUE THREAD

**Issue by brunano21** [Shouqun/node-dbus#71]
Hello,

I have a question about the module and the method someInterface.on(......).
In particular, I have 3 modules, let me call them A, B and C. In the module A, I require B and C, and these last two module use your module, dbus (getting the system bus).
In B module I get the following interface:
bus.getInterface('org.neard', '/', 'org.freedesktop.DBus.ObjectManager', function(err, iface)) {
iface.on('InterfacesAdded', function (ObjPath, dictionary) {
/* DO STUFF*/
console.log(ObjPath);
});
});

In C module, I get the following interface:
bus.getInterface('org.bluez', '/', 'org.freedesktop.DBus.ObjectManager', function(err, iface)) {
iface.on('InterfacesAdded', function (ObjPath, dictionary) {
/* DO STUFF*/
console.log(ObjPath);
});
)};

So basically the former is listening for InterfacesAdded coming from org.neard, while the latter is listening for InterfacesAdded coming from org.bluez.
Now the problem consists in, running the A module with nodejs, it happens that in the iface.on() method of the B module I receive some stuff coming from bluez, while I should get only stuff coming from org.neard. Furthermore, even if I get in B module stuff coming from org.bluez, in C module nothing happens, I mean, it's like the iface.on() method does not catch the signal sent by the org.neard. I think it's very strange and I do not know how to fix that.
I hope you can give any suggestions about that.

Thank you!

**Comment by digitalsigil** [comment]
I see this same issue as well. With this code
```javascript
bus.getInterface(
    'org.mpris.MediaPlayer2.vlc',
    '/org/mpris/MediaPlayer2',
    'org.freedesktop.DBus.Properties',
    function (err, iface) {
	iface.on(
	    'PropertiesChanged',
	    propertiesChangedCallback);
    });
```
My callback gets triggered by both VLC and spotify which both have a "/org/mpris/MediaPlayer2" object implementing "org.freedesktop.DBus.Properties". Since the Interface object returned by "getInterface" is associated with a service (in this case, "org.mpris.MediaPlayer2.vlc") I don't think this behavior is correct.

A fix for this would be to include the connection's unique ID in the signalHash.
